src/FMI2/int.jl

```diff
@@ -930,7 +930,7 @@ end
 export fmi2GetFMUstate
 
 """
-    fmi2FreeFMUstate!(c::FMU2Component, state::fmi2FMUstate)
+    fmi2FreeFMUstate(c::FMU2Component, state::fmi2FMUstate)
```

Comment:
This is a breaking change, did you upstream test this (so check of FMI/FMIFlux still work?).
Why do you want to change this?

Reply:
I would say, this is not a breaking change, it is a docstring (although it meight break the FMI-Doc action, but that will be easier to fix afterwards)